manifests/install.pp

```diff
@@ -2,5 +2,9 @@
   package { 'postgrey':
     ensure => present;
   }
+
+  file { '/etc/postgrey':
```

Comment:
Isn't this created by your package? What OS / package are you using?

Reply:
Sorry, didn't see these comments last time around.

CentOS 7.
(s:tvs-modules b:master) [root@yoda tvs-modules]# rpm -qf /etc/postgrey
file /etc/postgrey is not owned by any package

Looks like no.

Reply:
In Centos/RHEL7, the postgrey package provided by epel uses /etc/postfix/ as base config dir.